app/src/main/java/org/apache/taverna/mobile/ui/login/LoginPresenter.java

```diff
@@ -76,16 +75,15 @@ public void onError(Throwable e) {
                     }
 
                     @Override
-                    public void onNext(User user) {
-
+                    public void onComplete() {
+                        getMvpView().showDashboardActivity();
+                        getMvpView().showProgressDialog(false);
```

Comment:
`onComplete()` get called in the last of the cycle so basically, we use this when we make list of request and want to check all of them went well or not. Please move 
`
getMvpView().showDashboardActivity();
getMvpView().showProgressDialog(false);
`
in onNext.

Reply:
When i am moving these to OnNext then it showing the activity.Its keep loading.